templates/login/login_core.html.twig

```diff
@@ -0,0 +1,267 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {% if displayGoogleSignin %}
+        <meta name="google-signin-client_id" content="{{ googleSigninClientID }}">
+    {% endif %}
+    <title>{{ title|text }} {{ "Login"|xl }}</title>
+    {% block head %}{% endblock %}
+    <script src="{{ webroot }}/interface/product_registration/product_registration_service.js?v={{ jsIncludes }}"></script>
+    <script src="{{ webroot }}/interface/product_registration/product_registration_controller.js?v={{ jsIncludes }}"></script>
+
+    <script type="text/javascript">
+        document.addEventListener('DOMContentLoaded', () => {
+            document.getElementById('authUser').focus();
+        });
+        $(function () {
+            init();
+
+            var productRegistrationController = new ProductRegistrationController();
+            productRegistrationController.getProductRegistrationStatus(function (err, data) {
+                if (err) { return; }
+
+                if (data.statusAsString === 'UNREGISTERED') {
+                    productRegistrationController.showProductRegistrationModal();
+                }
+            });
+        });
+
+        function transmit_form(element) {
+            // disable submit button to insert a notification of working
+            element.disabled = true;
+            // nothing fancy. mainly for mobile.
+            element.innerHTML = '<i class="fa fa-sync fa-spin"></i> ' + jsText({{ "Authenticating"|xlj }});
+                {% if hasSession %}
+                document.cookie = {{ cookieText|xlj }};
```

Comment:
cookieText has already been escaped and shouldn't be translated. So can remove the `|xlj`